### PR TITLE
Add osu!trivium Quiz 2021 announcement newspost

### PR DESCRIPTION
**Pull request by mangomizer**

### CODE REVIEW

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2021-11-10-osu!trivium-quiz-2021.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+### Is there any question that requires knowledge from participating in the previous year's quiz?
+
+No. There will be no advantage gained (other than experience) from participating in previous events - we aim to have a level playing field.
+
+### Why am I not allowed to ask peppy for answers?
+
+peppy is incredibly busy and unless you want osu!lazer to replace stable by 2037, it would be best to not flood his inbox with questions that he doesn't even know the answers to. 
+
+Fun fact: peppy ended up scripting a response to everyone who emailed him during last year's osu!trivium quiz.
```
[the remark this answers is not in the document]
Comment (mangomizer) [reply news/2021-11-10-osu!trivium-quiz-2021.md:45]: Actually it was a scripted response, not a automatic script.

File: news/2021-11-10-osu!trivium-quiz-2021.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### When will the answers be released?
+
+Once all the submissions have been evaluated, the answers will be livestreamed on Twitch. We expect this to take place on the weekend after the submission deadline, though this obviously depends on the volume of submissions.
```
[the remark this answers is not in the document]
Comment (mangomizer) [reply news/2021-11-10-osu!trivium-quiz-2021.md:49]: Unsure if use my own or osu!live.

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2021-11-10-osu!trivium-quiz-2021.md:49]: we can announce that on twitter when it's decided anyway

File: news/2021-11-10-osu!trivium-quiz-2021.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2021-11-10-osu!trivium-quiz-2021.md:45]:
```suggestion
Fun fact: peppy ended up writing a scripted response to everyone who emailed him during last year's osu!trivium quiz.
```

File: news/2021-11-10-osu!trivium-quiz-2021.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2021-11-10-osu!trivium-quiz-2021.md:49]:
```suggestion
Once all the submissions have been evaluated, the answers will be livestreamed on Twitch — be sure to keep a close eye on the [@osugame Twitter](https://twitter.com/osugame) so you don't miss it! We expect this to take place on the weekend after the submission deadline, though this obviously depends on the volume of submissions.
```

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**